readthedocs/urls.py

```diff
@@ -44,6 +48,10 @@
     url(r'^notifications/', include('readthedocs.notifications.urls')),
     # For redirects
     url(r'^builds/', include('readthedocs.builds.urls')),
+    # For testing the 404's with DEBUG on.
+    url(r'^404/$', handler404),
+    # For testing the 500's with DEBUG on.
+    url(r'^500/$', handler500),
```

Comment:
Yea, I thought about it, but it's useful to confirm in production in some cases.